genesis/engine/coupler.py

```diff
@@ -1,7 +1,7 @@
 from typing import TYPE_CHECKING
 import numpy as np
 import taichi as ti
-
+import torch
 import genesis as gs
```

Comment:
Can you fix import formatting / ordering while you are here. 

Here is the rule: You should first import standards libraries, skip line, then ubiquitous third-party libraries (torch, numpy, tqdm...etc), skip line, then random third-party libraries, skip line, then our own custom libraries, skip line, then relative genesis imports. For each group, make sure it is sorted alphabetically.